pkg/plugins/golang/v3/scaffolds/internal/templates/makefile.go

```diff
@@ -84,7 +90,7 @@ manager: generate fmt vet
 
 # Run against the configured Kubernetes cluster in ~/.kube/config
 run: generate fmt vet manifests
-	go run ./main.go
+	go run ./main.go --config=${CONFIG_FILE}
```

Comment:
I still think this should be removed, since a user can more easily hard-code their local config path here than set an environment variable every time they `make run`.

Reply:
Sorry, I swear I removed this, might have been on my other machine 🤦‍♂️